metricbeat/module/elasticsearch/_meta/docs.asciidoc

```diff
@@ -2,3 +2,4 @@
 
 This is the elasticsearch Module. The elasticsearch module contains a minimal set of metrics to enable monitoring of elasticsearch across multiple versions.
 
+For more detailed monitoring of elasticsearch use the https://www.elastic.co/guide/en/x-pack/current/xpack-monitoring.html[free x-pack monitoring].
```

Comment:
May I suggest saying something like "To monitor more Elasticsearch metrics, use our https://www.elastic.co/guide/en/x-pack/current/xpack-monitoring.html[X-Pack monitoring] which is available under a free basic license."